src/k8s-engine/wsl.ts

```diff
@@ -1540,27 +1546,32 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
   }
 
   requiresRestartReasons(cfg: Settings['kubernetes']): Promise<Record<string, K8s.RestartReason | undefined>> {
```

Comment:
Probably a jsdoc explaining that the wsl version of this function doesn't need to be `async` because it doesn't have to get the `wsl` analog of `limaConfig`.